livekit-rtc/livekit/rtc/room.py

```diff
@@ -80,9 +81,9 @@ class RoomOptions:
 class DataPacket:
     data: bytes
     kind: proto_room.DataPacketKind.ValueType
```

Comment:
Weird thing is that ValueType doesn't exist anymore due to the pyi generator but mypy is not complaining ? Seems like the type checking isn't working?

Reply:
Oh this is Room is not from livekit-protocol, it's from livekit/rtc/_proto which is still generated by mypy-protobuf.

Reply:
Just to double check, changed kind to something bad (i.e. `str`) and ran it again and it failed as expected: https://github.com/livekit/python-sdks/actions/runs/8362789419